UglyTicTacToe/UglyTicTacToe/Game.aspx.cs

```diff
@@ -0,0 +1,206 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.UI;
+using System.Web.UI.WebControls;
+
+namespace UglyTicTacToe
+{
+    public enum Player
+    {
+        None,
+        X,
+        O
+    }
+    public partial class Game : Base
+    {
+       
+
+        private static string TurnKey = "turn";
+        public bool Turn
+        {
+            get
+            {
+                if (ViewState[TurnKey] == null)
+                {
+                    return true;
+                }
+                else
+                {
+                    return (bool)ViewState[TurnKey];
+                }
+            }
+            set
+            {
+                ViewState[TurnKey] = value;
+            }
+        }
```

Comment:
Since this property is the same across both of your ASPX pages, you can push it down into your `Base` class to remove code duplication.